### PR TITLE
Implement/fake configuration

### PR DESCRIPTION
**Pull request by Pe5h4**

### CODE REVIEW

**Comment by Pe5h4** [comment]
@ateska This is the initial attempt to implement `dev` config and fake `userinfo` parameters to simulate logged user. 

The changes I made:

1] Implemented devConfig into the config service and reducer
2] Added devConfig to the webpack.dev and webpack.build (in build the devConfig is everytime an empty object (there was no way to leave it undefined, it would cause the crash of the app)
3] Implement dispatching of fake userinfo to the store in auth module

it works nice in dev as well in build mode.

Here is the dev example
fakeuserinfo

**Comment by ateska** [comment]
Nice! ... and needs to be documented well.

**Review by ateska — APPROVED** [review]

**Comment by Pe5h4** [comment]
@ateska I added basic documentation to `README.md` file of ASAB WebUI. The one which is more comprehensive is stored at our wiki page. Is this sufficient?

**Review by ateska — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -66,21 +66,87 @@ $ yarn install
 $ yarn start
 ```
 
+### Run with `DEV` configuration
+
```
Comment (ateska) [inline README.md:70]: I suggest to "start with Why" ... e.g. why do I need to use this, what is this good for.

**Review by ateska — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Pe5h4) [reply README.md:72]: Updated https://github.com/TeskaLabs/asab-webui/pull/89/commits/6b57473be85dc337d434240c105cb5c04d6c6a4a

Comment (ateska) [reply README.md:72]: Hmms, I anticipated a documenatation about usage of the fake userinfo.
This is good but very generic.

**Review by ateska — CHANGES_REQUESTED** [review]
As mentioned above, I miss a handy documentation with example, how to use fake userinfo.

**Comment by Pe5h4** [comment]
@ateska Since description about `how to use fake userinfo` is larger, I created a markdown file and put it to the `doc` folder. I added the reference to the markdown file in the README. Is this approach ok? I bit doubt about the content of the markdown, where usage of fake userinfo is described...Can you please take a look on it for a feedback? Thank you https://github.com/TeskaLabs/asab-webui/pull/89/commits/4852816361e65690c977e141fca7ff30da128c9e

**Review by ateska — APPROVED** [review]

**Comment by ateska** [comment]
The clarity of this will need to be validated with other devs.

**Pull request merged**